### ISSUE TITLE
"Can not reach"

### ISSUE THREAD

**Issue by survivor303** [bkw777/mainline#206]
Cannot check updates.

![image](https://user-images.githubusercontent.com/3097696/224565396-196acc51-6b62-4380-93ae-cf3ba431afe1.png)

**Comment by bkw777** [comment]
Apologies, until I fix it, please manually put this into config.json:

  "ppa_uri" : "https://kernel.ubuntu.com/~kernel-ppa/mainline/",

example
```
{
  "ppa_uri" : "https://kernel.ubuntu.com/~kernel-ppa/mainline/",
  "all_proxy" : "",
  "notify_major" : "false",
  "notify_minor" : "false",
  "hide_unstable" : "true",
  "previous_majors" : "0",
  "notify_interval_unit" : "0",
  "notify_interval_value" : "1",
  "connect_timeout_seconds" : "15",
  "concurrent_downloads" : "4",
  "window_width" : "812",
  "window_height" : "612",
  "window_x" : "686",
  "window_y" : "452"
}
```

**Comment by el-quique** [comment]
> Apologies, until I fix it, please manually put this into config.json:
> 
> "ppa_uri" : "https://kernel.ubuntu.com/~kernel-ppa/mainline/",
> 
> example
> 
> ```
> {
>   "ppa_uri" : "https://kernel.ubuntu.com/~kernel-ppa/mainline/",
>   "all_proxy" : "",
>   "notify_major" : "false",
>   "notify_minor" : "false",
>   "hide_unstable" : "true",
>   "previous_majors" : "0",
>   "notify_interval_unit" : "0",
>   "notify_interval_value" : "1",
>   "connect_timeout_seconds" : "15",
>   "concurrent_downloads" : "4",
>   "window_width" : "812",
>   "window_height" : "612",
>   "window_x" : "686",
>   "window_y" : "452"
> }
> ```

Thx

**Comment by bkw777** [comment]
fixed in: c69ecf7
